ray-operator/config/samples/ray-cluster.complete.yaml

```diff
@@ -133,3 +133,13 @@ spec:
           - name: ray-logs
             emptyDir: {}
 ######################status#################################
+  # annotations passed on for the Head Service
+  headServiceAnnotations: {}
+  # custom labels passed on for the Head Service
+  # custom labels should not overwrite 
```

Comment:
```suggestion
  # to avoid conflicts, custom labels should not start with either of the prefixes `app.kubernetenes.io/` or `ray.io/`
```

Reply:
(Because the particular labels set by the operator and kubernetes might change in the future but the prefixes probably won't.)

Reply:
Thank you for pointing this out! I will change it accordingly.